templates/components/form/form.jsx

```diff
@@ -47,7 +62,10 @@ var Form = React.createClass({
       this.props.passwordError = null;
```

Comment:
can you file a followup bug to have password use Flux messaging rather than props?